pages/common/http.md

```diff
@@ -11,9 +11,9 @@
 
 `http --print {{H|B|h|b|m|Hh|Hhb|...}} {{https://example.com}}`
 
-- Specify the HTTP method when sending a request:
+- Specify the HTTP method when sending a request and use a proxy to intercept the request:
 
-`http {{GET|POST|HEAD|PUT|PATCH|DELETE|...}} {{https://example.com}}`
+`http {{GET|POST|HEAD|PUT|PATCH|DELETE|...}} --proxy {{http|https|...}}:{{http://localhost:8080|socks5://localhost:9050|...}} {{https://example.com}}`
```

Comment:
```suggestion
`http {{GET|POST|HEAD|PUT|PATCH|DELETE|...}} --proxy {{target_protocol}}:{{http://localhost:8080|socks5://localhost:9050|...}} {{https://example.com}}`
```

Reply:
The option with http|https|... looks better, but can be confusing.

Reply:
The `target_protocol` can also be confusing as the user may think that the target is the proxy server itself. 

Since `http` tool supports HTTP/HTTPS only, I think this would be better

`{{http|https}}`